templates/submitCommentForm.php

```diff
@@ -0,0 +1,39 @@
+<?php include $_SERVER['DOCUMENT_ROOT'] . "/templates/include/header.php" ?>
+
+<h1>Post A Comment!</h1>
+<hr>
+
+<form action=<?php echo "index.php?action=submitComment&id=" . $_GET['id'] ?> method="post">
+    <input type="hidden" name="mapId" id="mapId" value=<?php echo $_GET['id'] ?>>
+    <ul>
+        <li>
+            <label for="author">Author</label>
+            <input type="text" name="author" id="author" required autofocus placeholder="Review Author" maxLength="255">
+        </li>
+        <li>
+            <label for="rating">Rating</label>
+            <select name="rating" id="rating">
+                <option value="0">No Rating</option>
+                <option value="1">One Star</option>
+                <option value="2">Two Stars</option>
+                <option value="3">Three Stars</option>
+                <option value="4">Four Stars</option>
+                <option value="5">Five Stars</option>
+            </select>
+        </li>
+        <li>
+            <label for="screenshotLink">Screenshot Link</label>
+            <input type="text" name="screenshotLink" id="screenshotLink">
+        </li>
+        <li>
+            <label for="comment">Comment</label>
+            <textarea name="comment" id="comment" required placeholder="Comment Contents" style="height: 15em;">None</textarea>
```

Comment:
Some bot is going to try to paste an entire novel into this field if there's no character limit, comments should probably have a limit that's quite a bit smaller than the limits on a map post.

Reply:
I've adjusted the limits on comments so they must be a minimum of 32 characters, and a maximum of 1024 characters. I've also added a limit to the screenshot links as well, they can be up to a maximum of 255 characters.

See commit `f5bb6c7` for the details.